core/sawtooth/cli/monitor.py

```diff
@@ -1,261 +0,0 @@
-# Copyright 2016 Intel Corporation
```

Comment:
What if we just look in 0.7 for this code? That's more preferable to me than having dead code in the repo.